### PR TITLE
[ONPREM-319] Add manual machine agent start - DO NOT MERGE

### PR DESCRIPTION
**Pull request by Stuart133**
# Description

Add a page describing how to manually start machine agent

# Reasons

https://circleci.atlassian.net/browse/ONPREM-319

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [x] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [x] Keep the title between 20 and 70 characters.
- [x] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [x] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/manually-starting-machine-runner.adoc [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+```script
+./circleci-runner machine -c /path/to/config.yaml
+```
+
+The agent will output logs to `stdout` indicating it has start correctly.
+
+[#run-a-job]
+== 3. Run a job
+
+You can now trigger a CircleCI job using the resource class associated with the machine runner agent. The agent will claim and run the task
```
Comment (ryanwohara) [inline jekyll/_cci2/manually-starting-machine-runner.adoc:62]:
```suggestion
You can now trigger a CircleCI job using the resource class associated with the machine runner agent. The agent will claim and run the task.
```

**Review by atulsingh0 — APPROVED** [review]

File: jekyll/_cci2/manually-starting-machine-runner.adoc [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  format: "text"
+  file: "-"
+```
+
+[#start-machine-runner-agent]
+== 2. Start the machine runner agent
+
+The following command is used to launch the machine runner agent:
+
+```script
```
[the remark this answers is not in the document]
Comment (Stuart133) [reply jekyll/_cci2/manually-starting-machine-runner.adoc:53]: Ahh it seems `shell` is the right one (looking at the other docs pages)

**Review by atulsingh0 — APPROVED** [review]

File: jekyll/_cci2/manually-starting-machine-runner.adoc [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+- Created a working directory for jobs
+
+[#download-runner-agent]
+== 1. Downloaded the runner agent binary
+
+The runner agent binary can be downloaded from S3 using the following commands:
+
+```shell
+curl -sfL $(curl -sfL https://circleci-binary-releases.s3.amazonaws.com/circleci-runner/manifest.json \
+  | jq -r .releases.current.linux.amd64.url) \
```
Comment (christian-stephen) [inline jekyll/_cci2/manually-starting-machine-runner.adoc:34]: Should we make these instructions arch agnostic?

File: jekyll/_cci2/manually-starting-machine-runner.adoc [hunk 4]
```diff
@@ -0,0 +1,80 @@
+---
+contentTags:
+  platform:
+  - Cloud
+  - Server v4.x
+  - Server v3.x
+---
+= Manually starting machine runner on Linux
```
Comment (rosieyohannan) [inline jekyll/_cci2/manually-starting-machine-runner.adoc:8]: Is this to take the place of the manual machine runner installation on Linux that we currently have here? https://github.com/circleci/circleci-docs/pull/8329/files#diff-23c3aec405f4050173b7a8d02816091669f2a51e840fc82514ab76ab3efef2a7

If so we need to look at the prerequisite steps we have listed there to keep things consistent. We're using reusable snippets quite a lot in these runner docs, and we would like to keep the language consistent through all pages.

I was a bit confused by the different language choice "manually start jobs using machine runner" As opposed to install machine runner?

**Comment by Stuart133** [comment]
Closing - We have decided to not explicitly document this process and instead have a reference page detailing the commands & options the agent uses

**Pull request closed without merging**